python/cudf/cudf/core/dataframe.py

```diff
@@ -6050,28 +6096,15 @@ def _reduce(
         self,
         op,
         axis=None,
-        level=None,
-        numeric_only=None,
+        numeric_only=False,
         **kwargs,
     ):
-        if level is not None:
-            raise NotImplementedError("level parameter is not implemented yet")
-
         source = self
-        if numeric_only:
-            numeric_cols = (
-                name
-                for name in self._data.names
-                if is_numeric_dtype(self._data[name].dtype)
-            )
-            source = self._get_columns_by_label(numeric_cols)
-            if source.empty:
-                return Series(index=cudf.Index([], dtype="str"))
 
         if axis is None:
-            if op in {"any", "all"}:
-                axis = 2
-            else:
+            # if op in {"any", "all"}:
+            #     axis = 2
```

Comment:
Is this meant to be commented out? Should there be a TODO here?

Reply:
Stale, removed it.